tests/unit/test_parsers.py

```diff
@@ -794,6 +826,10 @@ def test_parsers(parser_class, raw_file, results_file):
     with open(raw_file, "rb") as file_obj:
         raw_data = file_obj.read()
 
+    if issubclass(parser_class, Xlsx) and not READ_EXCEL_PRESENT:
```

Comment:
same as before, should'nt we install all needed for testing?

Reply:
I can do that locally, but Github CI is not installing all packages.  This is how I worked around that.